documentation/modules/exploit/linux/http/kaltura_unserialize_cookie_rce.md

```diff
@@ -0,0 +1,46 @@
+## Description 
+
+The getUserzoneCookie function in Kaltura before 13.2.0 uses a hardcoded cookie secret to validate cookie signatures, which allows remote attackers to bypass an intended protection mechanism and consequently conduct PHP object injection attacks and execute arbitrary PHP code via a crafted userzone cookie. 
+ 
+
+## Vulnerable Application
+
+ This module exploits a remote code execution within the Kaltura(<=13.1.0) via a cookie deserialization.
+ Vulnerability reference- https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2017-14143.
+
+
+## Verification Steps
+
+ 1. Start msfconsole
+ 2. use exploit/linux/http/kaltura_unserialize_cookie_rce
+ 3. set RHOST https://example.com (or IP address)
+ 4. set ENTRYID 0_xxxxxxxx
+ 5. set payload generic/custom
+ 6. set payloadstr "system('command you want to execute, eg.- ls -la');"
+ 7. run
+
+
+## Options
+
+ default RPORT 4444
+
+
+## Scenarios
+
```

Comment:
Typically we like a 3 # and then info about the scenario.  Is this 13.1? 12.0? I see its linux and most likely debian/ubuntu, its nice to know the actual system.
Example: ```### Kaltura 13.0 on Ubuntu 16.04 x64```